.github/workflows/cron-job-its.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     needs: build
     with:
       build_jdk: 17
-      runtime_jdk: 21.0.4
+      runtime_jdk: 17
```

Comment:
@Akshat-Jain, looks like there's one more place where we use  jdk 21.0.4 with jdk 17 in the [revised its](https://github.com/apache/druid/blob/master/.github/workflows/revised-its.yml#L90). Should we change that as well? All the other places in the revised-its seem to use build_jdk17/runtime_jdk 17.

Reply:
@abhishekrb19 Have made the change.
That wasn't affecting us because `runtime_jdk` is unused in `reusable-revised-its.yml`, but that's a separate issue and has been there since inception.